test/integration/webhook/jobs/suite_test.go

```diff
@@ -71,6 +73,10 @@ var _ = ginkgo.AfterSuite(func() {
 
 func managerSetup(setup func(ctrl.Manager, ...jobframework.Option) error, opts ...jobframework.Option) framework.ManagerSetup {
 	return func(ctx context.Context, mgr manager.Manager) {
+		cCache := cache.New(mgr.GetClient())
+		queues := queue.NewManager(mgr.GetClient(), cCache)
+		opts = append(opts, jobframework.WithCache(cCache), jobframework.WithQueues(queues))
+
```

Comment:
What is the reason for the initialization of the cache and queue?

Reply:
Wit MK enabled (by default) the defaulting webhook is now hitting:

https://github.com/kubernetes-sigs/kueue/blob/6cace04a628195f52fad302e10127cdbbad6a380/pkg/controller/jobs/jobset/jobset_webhook.go#L72-L89

so we need to provide a cache and a queue manager.

Reply:
sgtm

Reply:
Ah, that makes sense.
Thank you for the clarifications!